### DISCUSSION TITLE
Unable to `create` with mongodb driver

### DISCUSSION THREAD

**Discussion by quinnjr** [prisma/prisma#12747] [Q&A] [answered]
Hello,

I seem to be having a weird issue with the MongoDB driver where I can't create records in my database. The Rust code complains that `create` is not yet implemented.

```sh
thread 'tokio-runtime-worker' panicked at 'not yet implemented', query-engine/connectors/mongodb-query-connector/src/logger.rs:160:31
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
PrismaClientUnknownRequestError:
Invalid `prisma.plugin.create()` invocation in
/app/prisma/seed.ts:73:25

   70   }
   71 };
   72
→  73 await prisma.plugin.create(
  not yet implemented
    at Object.request (/app/node_modules/.pnpm/@prisma+client@3.12.0_prisma@3.12.0/node_modules/@prisma/client/runtime/index.js:45582:15)
    at async PrismaClient._request (/app/node_modules/.pnpm/@prisma+client@3.12.0_prisma@3.12.0/node_modules/@prisma/client/runtime/index.js:46405:18)
    at async seed (/app/prisma/seed.ts:73:5) {
  clientVersion: '3.12.0'
}
```

My code that I'm trying to execute in a database seed:

```typescript
  for (let plugin of Plugins) {
    const category = await prisma.category.findUnique({
      where: {
        name: plugin.category
      }
    });

    if (!category || !category.id) {
      console.log(`error: ${JSON.stringify(plugin)} ${category}`);
    }

    plugin.category = {
      connect: {
        id: category!.id
      }
    };

    const language = await prisma.language.findFirst({
      where: {
        name: {
          contains: plugin.language
        }
      }
    });

    if (!language || !language.id) {
      console.log('error: ' + plugin.language);
    }

    plugin.language = {
      connect: {
        id: language!.id
      }
    };

    await prisma.plugin.create({
      data: plugin
    });
  }
```

Thanks for your help!

**Comment by alex-semenyuk** [comment, marked as answer]
I believe, it's known issue https://github.com/prisma/prisma/issues/12708 with hight priority

**Reply by janpio** [reply]
The issue you linked to is only about the crash reporting.

The actual problem is here: https://github.com/prisma/prisma/issues/12704

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
